Source/ASPagerNode.m

```diff
@@ -148,7 +148,7 @@ - (NSInteger)indexOfPageWithNode:(ASCellNode *)node
 
 #pragma mark - ASCollectionGalleryLayoutPropertiesProviding
 
-- (CGSize)sizeForElements:(ASElementMap *)elements
+- (CGSize)galleryLayoutDelegate:(nonnull ASCollectionGalleryLayoutDelegate *)delegate sizeForElements:(nonnull ASElementMap *)elements
```

Comment:
Will this be a breaking API change or do we see the `ASCollectionGalleryLayout` as beta or not announced yet?

Reply:
Yes, the whole async collection layout API is still in beta.